components/RichTextEditor.js

```diff
@@ -419,12 +427,29 @@ export default class RichTextEditor extends React.Component {
         )}
         <input id={this.state.id} value={value || defaultValue} type="hidden" name={inputName} />
         <HTMLContent fontSize={fontSize}>
-          <trix-editor
-            ref={this.editorRef}
-            input={this.state.id}
-            autofocus={autoFocus ? true : undefined}
-            placeholder={placeholder}
-          />
+          {!showCount ? (
+            <trix-editor
+              ref={this.editorRef}
+              input={this.state.id}
+              autofocus={autoFocus ? true : undefined}
+              placeholder={placeholder}
+            />
+          ) : (
+            <Container position="relative">
+              <trix-editor
+                ref={this.editorRef}
+                input={this.state.id}
+                autofocus={autoFocus ? true : undefined}
+                placeholder={placeholder}
+              />
+              <Container position="absolute" bottom="1em" right="1em">
+                <StyledTag textTransform="uppercase">
+                  <span>{defaultValue.length}</span>
```

Comment:
Good idea to display that here. However, the way we compute this length is misleading: if I type 10 characters in bold it should display `10` (the length without HTML).

Reply:
So, do we have any util to strip html?

Reply:
I just added one